qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst

```diff
@@ -0,0 +1,28 @@
+.. _ebi-submission:
+
+.. index:: ebi-submission
+
+EBI submission via Qiita
+========================
+
+Qiita allows users to deposit their study, sample, experiment and sequence data to the
+`European Nucleotide Archive (ENA) <https://www.ebi.ac.uk/ena>`__, which is a permanent repository
+part of the `European Bioinformatics Institute (EBI) <https://www.ebi.ac.uk/>`__. Submitting to
+this repository will provide you with a unique identifier for your study, which is generally a
+requirement for publication.
+
+EBI/ENA requires a given set of column fields to describe your samples and experiments, for more
+information visit :doc:`prepare-templates` and pay most attention to EBI required fields,
+without these Qiita Admins will not be able to submit. If you want to submit your data or need
+help send an email to `qiita.help@gmail.com <qiita.help@gmail.com>`__.
+
+Note that this kind of submissions are time intensive and need full collaboration from the user.
+Thus, do not wait until the last minute to request help. In general, the best time to request a submission
+is when you are writing your paper. Remember that the data can be submitted to EBI and can be
+kept private and simply make public when the paper is accepted. Note that EBI/ENA takes up to 15 days to
+change the status from private to public, so consider this when submitting data and your manuscript.
+
+For convenience Qiita allows you to upload a QIIME mapping file to process your data. However,
```

Comment:
Sounds excellent, thanks for trying them out :D

On (Dec-18-15| 5:28), Antonio Gonzalez wrote:
> > +part of the `European Bioinformatics Institute (EBI) <https://www.ebi.ac.uk/>`__. Submitting to
> > +this repository will provide you with a unique identifier for your study, which is generally a
> > +requirement for publication.
> > +
> > +EBI/ENA requires a given set of column fields to describe your samples and experiments, for more
> > +information visit :doc:`prepare-templates` and pay most attention to EBI required fields,
> > +without these Qiita Admins will not be able to submit. If you want to submit your data or need
> > +help send an email to `qiita.help@gmail.com <qiita.help@gmail.com>`__.
> > +
> > +Note that this kind of submissions are time intensive and need full collaboration from the user.
> > +Thus, do not wait until the last minute to request help. In general, the best time to request a submission
> > +is when you are writing your paper. Remember that the data can be submitted to EBI and can be
> > +kept private and simply make public when the paper is accepted. Note that EBI/ENA takes up to 15 days to
> > +change the status from private to public, so consider this when submitting data and your manuscript.
> > +
> > +For convenience Qiita allows you to upload a QIIME mapping file to process your data. However,
> 
> However, after displaying it as an error I realized that it adds the header "Error", which is misleading, also tried warning and is also misleading. Thus, decided by note, which I liked.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1579/files#r48024380